src/index.js

```diff
@@ -2,6 +2,10 @@
 import * as React from "react";
 import * as PropTypes from "prop-types";
 
+export class EFX<S: {}> {
+  Store: Class<Store<S>>;
+}
+
```

Comment:
Classes in Flow 
1) can have type parameters that are scoped within the class body
2) have a first class representation both at runtime and at the type level (edited)

This allows us to implement ML functors; i.e. given a module, return a module closed over the same type variables. 

Translation: given a class, return a class closed over the same type variables